lightning/src/ln/channel.rs

```diff
@@ -3679,7 +3679,8 @@ impl<Signer: Sign> Channel<Signer> {
 	}
 
 	/// Indicates the funding transaction is no longer confirmed in the main chain. This may
-	/// force-close the channel, but may also indicate a harmless reorganization of a block or two.
+	/// force-close the channel, but may also indicate a harmless reorganization of a block or two
+	/// before the channel has reached funding_locked and we can just wait for more blocks.
```

Comment:
Second sentence is a bit wordy / run-on. But isn't it really the caller that will close the channel upon receiving an error back?